### ISSUE TITLE
fuyukuru crash because this method call is not implemented with the passed parameter combination

### ISSUE THREAD

**Issue by wuyudi** [doitsujin/dxvk#2992]
### Software information

http://www.silkysplus.jp/game/fuyukuru/index.html, it's a r-18 game which requires directx11

### System information
- GPU: AMD ATI **Cezanne**
- Driver: 05:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Cezanne (rev c2)
- Wine version: wine-7.18
- DXVK version: 1.10.3-1

### Apitrace file(s)
- Put a link here

This program crashes because of 
![image](https://user-images.githubusercontent.com/30024051/196059568-42ed8af8-af8d-416f-b553-70b76eb97304.png)

so only several KB.

[fuyukuru.trace.tar.gz](https://github.com/doitsujin/dxvk/files/9795737/fuyukuru.trace.tar.gz)

For instructions on how to use apitrace, see: https://github.com/doitsujin/dxvk/wiki/Using-Apitrace

### Log files
- d3d9.log: None
- d3d11.log:
[fuyukuru_d3d11.tar.gz](https://github.com/doitsujin/dxvk/files/9795754/fuyukuru_d3d11.tar.gz)

- dxgi.log:
[fuyukuru_dxgi.tar.gz](https://github.com/doitsujin/dxvk/files/9795755/fuyukuru_dxgi.tar.gz)

I run

```bash
LC_ALL=ja_JP.utf8 wine ./fuyukuru.exe &> log
```

and here is the log.

[log.tar.gz](https://github.com/doitsujin/dxvk/files/9795743/log.tar.gz)

**Comment by doitsujin** [comment]
Trying to open the trace crashes here:
> Loading: "fuyukuru.trace"
> error: (DXGISwapChain4::QueryInterfacethisriidppvObj) unknown call detail 65

Which apitrace build was this created with?

**Comment by doitsujin** [comment]
The trial version of that game from the website works fine here (using current DXVK master) and some old wine-tkg 7.16 build.

Looking at your wine log though, this is probably a Direct**2**D issue:
> 0024:fixme:d2d:d2d_factory_ID2D1Factory2_CreateDevice iface 01590698, dxgi_device 00936300, device 0092D600 stub!

Since this works on my local wine build, I'm guessing that this currently requires wine-staging or perhaps Proton patches to run.

**Comment by wuyudi** [comment]
> Trying to open the trace crashes here:
> 
> > Loading: "fuyukuru.trace"
> > error: (DXGISwapChain4::QueryInterfacethisriidppvObj) unknown call detail 65
> 
> Which apitrace build was this created with?

```bash
file fuyukuru.exe
fuyukuru.exe: PE32 executable (GUI) Intel 80386, for MS Windows
```

So I use https://github.com/apitrace/apitrace/releases/download/11.1/apitrace-11.1-win32.7z

and put those four dlls in the same folder.

**Comment by wuyudi** [comment]
> The trial version of that game from the website works fine here (using current DXVK master) and some old wine-tkg 7.16 build.
> 
> Looking at your wine log though, this is probably a Direct**2**D issue:
> 
> > 0024:fixme:d2d:d2d_factory_ID2D1Factory2_CreateDevice iface 01590698, dxgi_device 00936300, device 0092D600 stub!
> 
> Since this works on my local wine build, I'm guessing that this currently requires wine-staging or perhaps Proton patches to run.

I just tried it... so strange, 
![image](https://user-images.githubusercontent.com/30024051/196096262-cc319ddb-14c3-42b9-96cd-f87132fc7472.png)

It works..

[log.tar.gz](https://github.com/doitsujin/dxvk/files/9797160/log.tar.gz)

But the full version not.

**Comment by wuyudi** [comment]
I used VirtualBox to launch win10 and tested it, both trial and full version works well.

Then I found that there are some missing pictures.

For example, click the "はじめから"(start the game from beginning) at title page.

there are some sentences missing from wine.(Though I guess it's not the reason wine not work)

(正在运行 means "it's running")

![image](https://user-images.githubusercontent.com/30024051/197182393-c7ef246c-908b-48e1-aafe-dc47d871d03f.png)
